src/Redux/Selectors/MountedComponentsSelectors.js

```diff
@@ -0,0 +1 @@
+export const isMountedComponentSelector = name => state => state.MountedComponents[name];
```

Comment:
also - same question about exporting as in `ActionsSelector`.

Reply:
might be another selectors in the future.
if it doesn't necessary for now we can put it as default.